syntax = "proto3";

message SendEntry {
	int32 to = 1;
	int64 seq = 2;
	int64 t_sent = 3;
}

message ReceiveEntry {
	int32 origin = 1;
	int64 seq = 2;
	int64 t_sent = 3;
	int64 t_received = 4;
	int32 hops = 5;
}

message SendLog {
	repeated SendEntry entries = 1;
}

message ReceiveLog {
	repeated ReceiveEntry entries = 1;
}
